public/images/hamburger.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" viewBox="0 0 384 384">
+<g>
+  <rect x="0" y="277.333" width="384" height="42.667" fill="#728448"/>
+  <rect x="0" y="170.667" width="384" height="42.667" fill="#728448"/>
+  <rect x="0" y="64" width="384" height="42.667" fill="#728448"/>
+</g>
+</svg>
```

Comment:
There's need to design new icon for hamburger with site colors.

Reply:
Let's use same color as desktop version of language selector.

Reply:
Which files you have in mind? If it's a simple icon, then yes, vector graphic is smaller than rasterized. But if it's an image, you can't easily convert to SVG.